GETTING_STARTED.md

```diff
@@ -79,6 +80,15 @@ show_result(img, result, model.CLASSES)
 imgs = ['test1.jpg', 'test2.jpg']
 for i, result in enumerate(inference_detector(model, imgs)):
     show_result(imgs[i], result, model.CLASSES, out_file='result_{}.jpg'.format(i))
+
+# test a video and show the results
+cap = cv2.VideoCapture('video.mp4')
```

Comment:
It is simpler to use [`mmcv.VideoReader`](https://mmcv.readthedocs.io/en/latest/video.html#videoreader).